ql/experimental/exoticoptions/analyticholderextensibleoptionengine.cpp

```diff
@@ -65,7 +65,7 @@ namespace QuantLib {
         //calculate payoff discount factor assuming continuous compounding
         DiscountFactor discount = riskFreeDiscount(t1);
         Real result = 0;
-        constexpr Real minusInf = -std::numeric_limits<Real>::infinity();
+        const Real minusInf = -std::numeric_limits<Real>::infinity();
```

Comment:
These and the others are constants, so I'd use `double` as you did in the change right above.  What do you think?

Reply:
We thought about that, but there could be an issue with this in the remote (admittedly unlikely) case that `Real` is something else than `double` with a different representation of infinity. If we define a constant with double-infinity, and then use it in calculations with `Real`, it may become an issue. 

The case above is different, as it's just a regular number, not infinity.

Reply:
I'd keep the double constants now and think about the unlikely case if it comes up.